apps/game/lib/game/bid.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Game.Bid do
+
+  defstruct dice_count: 1, face_value: 2
+
+  alias Game.Bid
+
+  @spec valid?(%Bid{}, %Bid{}, integer) :: boolean
+  def valid?(new_bid, current_bid, dice_count) do
+    has_valid_face_value?(new_bid) &&
```

Comment:
I think the `&&` version will bail at the first failure, whereas the Enum would have to calculate them all first?